jax/_src/test_util.py

```diff
@@ -1746,6 +1746,22 @@ def sqrt(self, x):
   def expm1(self, x):
     return x.context.expm1(x)
 
+  def log1p(self, x):
+    ctx = x.context
+    # workaround mpmath bugs:
```

Comment:
You should link the bug here in the form of `TODO(pearu): remove workaround after mpmath bug <link> is fixed`, so it's obvious when this can be removed.

Reply:
Done.